docs/shared/metrics/prometheus-exemplar-support.adoc

```diff
@@ -0,0 +1,99 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:description: Helidon metrics
+:keywords: helidon, metrics, exemplar, prometheus, OpenMetrics
+:zipkin-page: ../../shared/tracing/tracer-zipkin.adoc
+:jaeger-page: ../../shared/tracing/tracer-jaeger.adoc
+:open-metrics-spec-exemplars: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#exemplars
+
+Add Helidon {h1Prefix} support for  link:{open-metrics-spec-exemplars}[OpenMetrics (Prometheus) exemplars] for histograms, counters, and simple timers to your application simply by adding dependencies to your project's `pom.xml`.
+
+== Prerequisites
+
+Declare the following dependency in your project:
+
+[source,xml,subs="verbatim,attributes"]
+----
+<dependency>
+    <groupId>io.helidon.metrics</groupId>
+    <artifactId>helidon-metrics-trace-exemplar</artifactId>
+    <scope>runtime</scope>
+</dependency>
+----
+
+Also, include either link:{zipkin-page}[Helidon Zipkin] or link:{jaeger-page}[Helidon Jaeger] support:
+include::{zipkin-page}[tag=zipkin-dependency]
+or
+include::{jaeger-page}[tag=jaeger-dependency]
+
+Be sure Zipkin or Jaeger, whichever you chose, is running and accessible to your server.
+
+== Interpreting Exemplars
+
+//[quote,Mirriam-Webster Dictionary,'link:https://www.merriam-webster.com/dictionary/exemplar[exemplar]'']
+//____
+//exemplar - one that serves as a model or example
+//____
+[NOTE]
+--
+link:https://www.merriam-webster.com/dictionary/exemplar[_exemplar_] - one that serves as a model or example
+[.text-right]
+-- Merriam-Webster Dictionary
+--
+
+When you add the `helidon-metrics-trace-exemplar` dependency--and one for either Zipkin or Jaeger--to your application, Helidon automatically records a label and timestamp with each histogram and simple timer sample and with each counter update. Helidon adds the label, sample value, and timestamp to the OpenMetrics output returned from the Helidon metrics endpoint (`/metrics` unless you set it up otherwise).
```

Comment:
with each histogram, and a simple timer sample with each counter update.

Reply:
@ljamen I'll plan to change it to this: "..., Helidon automatically records a sample (label, value, and timestamp) with each update to a histogram, simple timer, or counter."